README.md

```diff
@@ -289,3 +289,44 @@ RETURNS:
 To check if your POST was successful, follow the steps in [registering a
 manifestation](#was-my-post-to-manifestations-successful) and use the returned
 Right's data instead.
+
+
+### Transfer a Right
+
+You may only transfer a Right that you are currently holding. RightsAssignment
+entities are automatically created for each transfer and may include additional,
+arbitrary attributes if a `rightsAssignment` dict is given in the payload.
+
+```
+POST /api/v1/rights/transfer
+HEADERS {"Content-Type": "application/json"}
+
+PAYLOAD:
+{
+    "rightId": "<ID of an existing Right to transfer; must be held by the user specified in `currentHolder`>",
+    "rightsAssignment": {
+        ...
+    },
+    "currentHolder": {
+        "verifyingKey": "<base58 string>",
+        "signingKey": "<base58 string>"
+    },
+    "to": {
+        "verifyingKey": "<base58 string>",
+        "signingKey": "<base58 string>"
```

Comment:
We can omit the `signingKey` since it's never used. I'd prefer to keep it in the dict, but set to null, so that the payload still matches `pycoalaip-bigchaindb`'s user model.